colors/green.md

```diff
@@ -3,29 +3,24 @@
 > Nature does not hurry, yet everything is accomplished.
 > <cite>Lao Tzu</cite>
 
-Green is the color of growth, life, and nature. It is the color of the forest and the wild, of animals, insects, plants, and druids. 
+Green is the color of growth, life, and nature. It is the color of the forest and the wild, of animals, insects, plants, and druids.
 
 ![Invigorate](https://s3.amazonaws.com/assets1.orbsccg.com/prod/cards/art/91X.jpg)
 
 ## Example Green Cards
 
 [[Wild Entomologist]]
 
-Wild Entomologist is a green minion that can be used to create an Insect minion token. If you get enough Insect minion
-tokens, you might be able to use them to swarm your opponent.
+Wild Entomologist is a green minion that can be used to create an Insect minion token. If you get enough Insect minion tokens, you might be able to use them to swarm your opponent.
 
 [[The Orb of the Pack]]
 
-The Orb of the Pack is a card that could work well with Insect token makers like Wild Entymologist. It gives all your minions
-a buff based on how many other minions with the subtype you have in play. If you have a lot of Insect minions, you can use this
-to make them all vastly stronger.
+The Orb of the Pack is a card that could work well with Insect token makers like Wild Entymologist. It gives all your minions a buff based on how many other minions with the subtype you have in play. If you have a lot of Insect minions, you can use this to make them all vastly stronger.
 
 [[Natural Force]]
 
-Natural Force is a green action card that can be used to destroy constructs or secrets. If you are playing against a deck that
-doesn't have constructs or secrets, you can energize it instead.
+Natural Force is a green action card that can be used to [destroy](../rules/glossary.md#destroy) constructs or [secrets](../rules/secrets.md). If you are playing against a deck that doesn't have constructs or secrets, you can energize it instead.
 
 [[Invigorate]]
 
-Invigorate is a green action card that gives your minion +2/+2 and Overwhelming temporarily. You can use this as a trick
-to save your minion from dying in battle, or to beat a blocking minion, or to push through extra damage to your opponent's life total.
+Invigorate is a green action card that gives your minion +2/+2 and [Overwhelming](../rules/glossary.md#overwhelming) temporarily. You can use this as a trick to save your minion from dying in battle, or to beat a blocking minion, or to push through extra damage to your opponent's life total.
```

Comment:
We can try this but I think the urls should just be like this in order for it to get rendered correctly on the website:
 [Overwhelming](/wiki/rules/glossary#overwhelming)
 
 Also I don't have the markdown automatically adding tags on the website yet but I think I can add that easily.

Reply:
Actually your URL style worked too https://orbsccg.com/wiki/colors/green